packs/data/pfs-season-2-bestiary.db/kareida.json

```diff
@@ -596,7 +596,29 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "key": "Aura",
+                        "predicate": [
+                            "scarecrow-mask"
+                        ],
+                        "radius": 30,
+                        "slug": "masked-intentions",
+                        "traits": [
+                            "aura",
+                            "mental",
+                            "visual"
+                        ]
+                    },
+                    {
+                        "domain": "all",
+                        "key": "RollOption",
+                        "label": "PF2E.NPCAbility.KareidaScarecrowMask",
```

Comment:
```suggestion
```
Don't need to localize this, it's PFS so just the ability name is fine rather than adding more to re-en.json.